interface/interface.dm

```diff
@@ -4,8 +4,12 @@
 	set desc = "Type what you want to know about.  This will open the wiki in your web browser."
 	set hidden = 1
 	if(config.wikiurl)
-		if(alert("Open the wiki in your browser?", null, "Yes", "No") == "Yes")
+		var/query = stripped_input(src, "Enter Search:", "Wiki Search", "Homepage")
```

Comment:
```suggestion
		var/query = stripped_input(src, "Enter Search. Leave the box blank to cancel", "Wiki Search", "Homepage")
```

Reply:
That's not how it works - there's an actual "cancel" button.